src/js/spreadsheet-events.js

```diff
@@ -83,21 +94,39 @@ RiseVision.SpreadsheetEvents = (function() {
         date.textContent = events[i].date;
         details.textContent = events[i].details;
 
-        slide.innerHTML = "<li class ='title'>" + title.textContent + "</li>" + "<li class ='date'>" + date.textContent + "</li>" + "<li class ='details'>" + details.textContent + "</li>"
+        mainSlide.innerHTML = "<li class ='title'>" + title.textContent + "</li>" + "<li class ='date'>" + date.textContent + "</li>" + "<li class ='details'>" + details.textContent + "</li>";
+
+        $("#slideArea").append(mainSlide);
       }
+
+      eventSlides = document.getElementsByClassName("slide");
+
+      eventSlides[0].style.right = "0vw";
+
+        function changeSlide() {
+          slideMovement = true;
+          $(eventSlides[counter]).animate({ "right": "0" }, {duration: 1000, easing: "easeOutBack"}).delay(5000).animate({ "right": "100vw" }, 1000);
+          counter++;
+          if (counter >= numEvents) {
+            counter = 0;
+          }
+        }
+        slideTimer = setInterval(changeSlide, 7000);
     }
 
   /*
    *  Public Methods
    */
-  function init(e) {
+  function init() {
     var googleSheet = document.getElementById("googleSheet");
-
     googleSheet.addEventListener("rise-google-sheet-response", function(e) {
       addEvents(e.detail.cells);
       displayEvents();
+      counter = 0;
+      if (slideMovement === true) {
```

Comment:
@pcsandford Curious, if you're clearing the interval here, does it ever need to be started again?